middlewares/accesslog/json_log_formatter.go

```diff
@@ -0,0 +1,235 @@
+package accesslog
```

Comment:
It was written by one of my colleagues, rather than me personally, not copied from anywhere else.